openingd/opening.c

```diff
@@ -261,6 +262,8 @@ static u8 *funder_channel(struct state *state,
 
 	state->remoteconf = tal(state, struct channel_config);
 
+	peer_billboard(false,
```

Comment:
Suggest:

    #define now_doing(what, ...) peer_billboard(false, what, VA_ARGS)

Then:

    now_doing("Funding channel: offered, now waiting for accept_channel");